fedimint-cli/src/lib.rs

```diff
@@ -390,12 +390,19 @@ enum AdminCmd {
 
 #[derive(Debug, Clone, Subcommand)]
 enum DevCmd {
+    #[rustfmt::skip]
     /// Send direct method call to the API. If you specify --peer-id, it will
-    /// just ask one server, otherwise it will get consensus from all servers
+    /// just ask one server, otherwise it will try to get consensus from all servers.
+    ///
+    /// Example:
+    ///
+    /// fedimint-cli dev api --peer-id config '"fed114znk7uk7ppugdjuytr8venqf2tkywd65cqvg3u93um64tu5cw4yr0n3fvn7qmwvm4g48cpndgnm4gqq4waen5te0xyerwt3s9cczuvf6xyurzde597s7crdvsk2vmyarjw9gwyqjdzj"'
```

Comment:
Isn't this missing the actual peer id? Also, this will look horrible when rendered since it's not in a monospace code block.

Reply:
https://github.com/fedimint/fedimint/pull/3237